load-testing/tests/relays_stress_test.go

```diff
@@ -511,20 +510,30 @@ func (s *relaysSuite) ALoadOfConcurrentRelayRequestsAreSentFromTheApplications()
 	<-s.ctx.Done()
 }
 
+func (s *relaysSuite) TheUserShouldWaitForSeconds(dur int64) {
+	time.Sleep(time.Duration(dur) * time.Second)
+}
+
```

Comment:
It looks like the step in the .feature file was reverted. Do we need the step definition still?